app/vep/utils/vcf_results.py

```diff
@@ -119,9 +121,9 @@ def _get_alt_allele_details(
             consequences.append(
                 model.PredictedTranscriptConsequence(
                     feature_type=model.FeatureType.transcript,
-                    stable_id=_get_csq_value(csq_values, "Feature", None, index_map),
-                    gene_stable_id=_get_csq_value(csq_values, "Gene", None, index_map),
-                    biotype=_get_csq_value(csq_values, "BIOTYPE", None, index_map),
+                    stable_id=_get_csq_value(csq_values, "Feature", "", index_map),
+                    gene_stable_id=_get_csq_value(csq_values, "Gene", "", index_map),
+                    biotype=_get_csq_value(csq_values, "BIOTYPE", "", index_map),
                     is_canonical=is_cononical,
```

Comment:
`is_cononical ` — check where this variable is used, and change `cononical` to `canonical`